cloud/pkg/admissioncontroller/admission.go

```diff
@@ -195,3 +231,58 @@ func registerValidateWebhook(client admissionregistrationv1beta1client.Validatin
 	}
 	return nil
 }
+
+// admitFunc is the type we use for all of our validators and mutators
+type admitFunc func(admissionv1beta1.AdmissionReview) *admissionv1beta1.AdmissionResponse
+
+func serve(w http.ResponseWriter, r *http.Request, admit admitFunc) {
+	var body []byte
+	if r.Body != nil {
+		if data, err := ioutil.ReadAll(r.Body); err == nil {
+			body = data
+		}
+	}
```

Comment:
should we return if ``` r.Body == nil ```

Reply:
I think so, thanks! will address this soon.